src/stratum_server.cpp

```diff
@@ -914,4 +917,75 @@ bool StratumServer::StratumClient::process_submit(rapidjson::Document& doc, uint
 	return static_cast<StratumServer*>(m_owner)->on_submit(this, id, job_id.GetString(), nonce.GetString(), result.GetString());
 }
 
+void StratumServer::api_update_local_stats()
+{
+	if (!m_pool->m_api) {
+		return;
+	}
+
+	if (!m_pool->params().m_localStats) {
+		return;
+	}
+
```

Comment:
I would put rate-limiter here (no more than 1 call per minute).

Reply:
Is a rate limiter required here if the `api_update_local_stats()` call is rate limited?